CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## \[Unreleased\]
+## [Unreleased]
```

Comment:
will https://github.com/firecracker-microvm/firecracker/blob/main/tools/release-prepare.sh#L57 keep working with this change?

Reply:
fixed!